.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "vendors/libgsm"]
 	path = vendors/libgsm
 	url = https://github.com/timothytylee/libgsm.git
+[submodule "vendors/pybind11"]
```

Comment:
Apparently you can [pip install pybind11](https://pybind11.readthedocs.io/en/stable/installing.html#include-with-pypi)